_posts/2017-04-04-things-the-rust-community-team-does.md

```diff
@@ -0,0 +1,101 @@
+
+## Watch the Forums
+
+We read and engage in the comment sections. First of all, we believe that the project should be engaged into discussions about it. Hat tip to Sri Ramkrishna for giving a whole talk about that subject: [Swimming with chum in shark infested water](https://www.youtube.com/watch?v=VYrL_l984-E). This engagement must be complex and assertive.
+
+Second, it is a great source of information about our project. We take appropriate action by either documenting these issues for the project or by getting people into a discussion with the people that are an expert in their subject.
+
+## Find Creative Solutions
+
+We come up with creative solutions to worldwide community issues, including marketing, funding, growth. (*shiver*) The interesting thing about doing these in a FOSS project is that they need to low- or zero-cost and time effective, which means many standard solutions don't apply.
+
+Marketing happens often in the sense that we think about how to get meaningful information out to people. A good example is that if you find us at a conference, you might also find one of edunhams handy [postcard with important links](https://github.com/rust-community/resources/blob/gh-pages/rust_printed-card.pdf) next to it.
+
+## General Promotion
+
+We help Rust projects at promotion. A recent example is [rust-bios survey](https://github.com/rust-bio/rust-bio/issues/119#issuecomment-290196421) for a grant application. Instead of having individuals promote themselves, we use our experience to help out, without requiring a formal request.
+
+## Meeting Organisation
+
+This isn't always done by the community team, but we do make sure that people can have a meeting with the core or other teams to talk about specific subjects. This mostly involves both finding interesting people to speak to, but also encouraging community members that it is definitely okay to try to get some of the other teams time.
+
+Standards are the Rust production user meetings, but there have also been meetings on other subjects, like graphics or embedded systems.
+
+## Identify Upcoming Issues
+
+This is also a bit of a fuzzy thing, but it is important to see problems coming early. In the best of situations, this allows us to start discussion problems without having someone take the effort to approach us.
+
+The community team also runs the yearly community survey, which helps here.
+
+## Find and Curate Learning Material
+
+We are constantly on the lookout for people that create learning material and encourage them to share it with the wider community. While these projects are not necessarily direct community team work, many of us have such projects. We also encourage translations of existing material.
+
+We set up and curate a project wide [YouTube Channel](https://www.youtube.com/channel/UCaYhcUwRBNscFNUKTjgPFiA). This gives people a one stop place to find video content, mostly meetup and conference talks.
+
+## Consulting
```

Comment:
Consulting doesn't quite seem like the right word for this section. I associate it more with commercial side of things that you and @carols10cents are doing.